internal/proto/controller/storage/credential/store/v1/credential.proto

```diff
@@ -58,3 +60,64 @@ message Dynamic {
   // @inject_tag: `gorm:"not_null"`
   string library_id = 2;
 }
+
+message StoreListQueryResult {
```

Comment:
Do we need this as a proto?

Reply:
Not necessarily, I just tried to follow the convention, I'm happy to replace this with a struct.